model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -45,20 +44,20 @@ class FindingsMatcherTest : WordSpec() {
 
     override fun isInstancePerTest() = true
 
-    private fun setupLicenseFinding(license: String, path: String, line: Int = Random.nextInt(1..1000)) {
+    private fun setupLicenseFinding(license: String, path: String, line: Int = 2) {
```

Comment:
I'm basically fine with dropping the random values to make the test fully deterministic. But we should not in turn add hard-coded / "magic" numbers as default arguments here and below, because it's unclear why e.g. 2 makes more sense as a default value than 1 here and below or vice versa. So I'd prefer to have no default values at all then, and always be explicit.